### ISSUE TITLE
gnatpp can’t find liblal_tools.dylib

### ISSUE THREAD

**Issue by simonjwright** [simonjwright/building-gcc-macos-native#11]
Recent builds (since 13.2.0-aarch64) don’t install `liblal_tools.dylib`. For example,
```
$ /opt/gcc-14.0.1-aarch64/bin/gnatpp
dyld[96538]: Library not loaded: @rpath/liblal_tools.dylib
  Referenced from: <572CF5A8-C8E3-3F65-B96E-1721900B64D4> /opt/gcc-14.0.1-aarch64/bin/gnatpp
  Reason: tried: '/opt/gcc-14.0.1-aarch64//lib/relocatable/liblal_tools.dylib' (no such file), ...
```

whereas the library’s actually
```
$ ls -l /opt/gcc-14.0.1-aarch64/lib/liblal_tools.dylib 
lrwxr-xr-x  1 simon  staff  47 20 Mar 14:11 /opt/gcc-14.0.1-aarch64/lib/liblal_tools.dylib -> ../lib/lal_tools.relocatable/liblal_tools.dylib
```

**Comment by simonjwright** [comment]
This seems to be related to the way I’ve taken to building: from `common.sh`,
```
# To keep the build away from the eventual target (this assumes the
# code is relocatable! which, now, it is.
PREFIX=$TOP/$ARCH/gcc-$VERSION-$ARCH
```

But why this should affect `gnatpp` and not e.g. `gprbuild` isn’t clear.

**Comment by simonjwright** [comment]
> But why this should affect `gnatpp` and not e.g. `gprbuild` isn’t clear.

`gprbuild` is linked statically. `gnatpp` & friends have to be linked dynamically because of [this issue](https://github.com/AdaCore/gprbuild/issues/97)

**Comment by simonjwright** [comment]
Fixed in the GCC 14.1.0 release.
